cmd/syncer/main.go

```diff
@@ -97,6 +100,30 @@ func main() {
 		*internalFSSName, *internalFSSNamespace, "", *operationMode)
 	admissionhandler.COInitParams = &syncer.COInitParams
 
+	// Disconnect VC session on restart
+	defer func() {
+		log.Info("cleaning up vc sessions")
+		if r := recover(); r != nil {
+			log.Error("panic", r)
```

Comment:
Can this be changed to log.Error("observed a panic and a restart was invoked", r)